src/createDataDictionary.m

```diff
@@ -2,29 +2,18 @@
 
 function createDataDictionary(cfg, logFile)
     %
-    % Short description of what the function does goes here.
+    % It creates the data dictionary to be associated with a _events.tsv file. It will create empty
```

Comment:
I think that in the long term this function should be able to write the dictionary for a wider variety of data files and not just events file. At least stim files too. Hopefully. :wink: